app/src/main/java/com/skydoves/disneycompose/ui/details/PosterDetails.kt

```diff
@@ -88,6 +87,8 @@ fun PosterDetails(
             }
             .padding(16.dp)
         )
+        // It doesn't work anymore
```

Comment:
Could you check the new version [1.0.5](https://github.com/skydoves/orchestra) works with this?